enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -575,10 +581,22 @@ func NewPebbleCache(env environment.Env, opts *Options) (*PebbleCache, error) {
 		includeMetadataSize:         opts.IncludeMetadataSize,
 	}
 
-	versionMetadata, err := pc.databaseVersionMetadata()
+	versionMetadata, err := pc.DatabaseVersionMetadata()
 	if err != nil {
 		return nil, err
 	}
+	if newlyCreated && *opts.ActiveKeyVersion < 0 {
```

Comment:
It would be a little neater if you write it like this:
```
	if newlyCreated && *opts.ActiveKeyVersion < 0 {
		activeVersion := *opts.ActiveKeyVersion
		if activeVersion < 0 {
			activeVersion = int64(filestore.MaxKeyVersion) - 1
		}
		versionMetadata.MinVersion = activeVersion
		versionMetadata.MaxVersion = activeVersion
		versionMetadata.LastModifyUsec = clock.Now().UnixMicro()
	}
```

Reply:
SG, done.